entrypoint.sh

```diff
@@ -16,12 +16,14 @@
 
 OPENSDS_HOTPOT_URL=${OPENSDS_HOTPOT_URL:-http://127.0.0.1:50040}
 OPENSDS_ORCHESTRATION_URL=${OPENSDS_ORCHESTRATION_URL:-http://127.0.0.1:5000}
+SODA_DELFIN_URL=${SODA_DELFIN_URL:-http://127.0.0.1:8190}
 OPENSDS_GELATO_URL=${OPENSDS_GELATO_URL:-http://127.0.0.1:8089}
 OPENSDS_S3_URL=${OPENSDS_S3_URL:-http://127.0.0.1:8090}
 OPENSDS_AUTH_URL=${OPENSDS_AUTH_URL:-http://127.0.0.1/identity}
 
 OPENSDS_HOTPOT_API_VERSION=${OPENSDS_HOTPOT_API_VERSION:-v1beta}
 OPENSDS_ORCHESTRATION_API_VERSION=${OPENSDS_ORCHESTRATION_API_VERSION:-orch}
+SODA_DELFIN_API_VERSION=${SODA_DELFIN_API_VERSION:-delfin}
```

Comment:
mention delfin here is for version default value? if so you can default it to v1

Reply:
This is a dashboard specific name resolution. Since the API version is same for Gelato and Delfin `v1` the dashboard code needs to be routed to the appropriate API. This is where the name comes in. the naming needs to be updated as they are not really API versions. I will make the appropriate changes